manager/integration/tests/test_migration.py

```diff
@@ -147,7 +146,47 @@ def test_migration_with_failed_replica(clients, volume_name):  # NOQA
        And the old failed replica will be cleaned up.
     8. Validate initially written test data.
     """
-    return
+    def finalizer():
+        exec_cmd = ["mkdir", "/var/lib/longhorn/replicas"]
+        subprocess.check_output(exec_cmd)
+
+    request.addfinalizer(finalizer)
+
+    client, volume, data = setup_migration_test(clients,
+                                                volume_name,
+                                                replica_cnt=3)
+
+    current_node = common.get_self_host_id()
+    hosts = get_hosts_for_migration_test(clients)
+    migrate_target = hosts[0]
+    volume.attach(hostId=current_node)
+    volume = common.wait_for_volume_healthy(client, volume_name)
+
+    exec_cmd = ["rm", "-rf",  "/var/lib/longhorn/replicas"]
+    subprocess.check_output(exec_cmd)
+    volume = common.wait_for_volume_degraded(client, volume_name)
+
+    volume.attach(hostId=migrate_target)
+
+    volume = common.wait_for_volume_migration_ready(client, volume_name)
+    for replica in volume.replicas:
+        if replica.hostId == current_node:
+            assert replica.running is False
+        else:
+            assert replica.running is True
```

Comment:
Make sure the current running replicas are newly created replicas.
You can verify this by recording the old replica list then checking if the new replica name is in the list

Reply:
All replicas will in running state except replica on current node. so use a count to check new running replica = 2